pkg/sync/v7/data/Deployment.apps/openshift-azure-logging/omsagent-rs.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: apps/v1
```

Comment:
i'm really not sure if this is  needed. In addition the daemonset version is better because that will continue to work as the size of the compute nodepool grows.

Reply:
I think Jobe said we needed it, it would be nice to get some clarification on why.

Reply:
Ok its been clarified. The deployment is a log analytics cluster-scoped agent while the daemonset is a log analytics node-scoped agent. They are both needed on every cluster.